### ISSUE TITLE
RMSNorm

### ISSUE THREAD

**Issue by jeremyfix** [jeremyfix/torchcvnn#34]
RMSNorm is like LayerNorm but without the centering https://pytorch.org/docs/stable/generated/torch.nn.modules.normalization.RMSNorm.html

Seems to work better than LayerNorm  ; Especially usefull for transformers.
